suites/nautilus/cephfs/tier-4_fs.yaml

```diff
@@ -0,0 +1,58 @@
+---
+#=======================================================================================================================
+# Tier-level: 2
```

Comment:
Minor... the tier level in the filename and the doc string is different.

Could we have a proper file name conveying the intent of what is going to be tested?

Reply:
Sure